src/calculateRentalCost.js

```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const costPerDay = days * 40;
```

Comment:
Ideally we would like to know what 40 is by assigning it to a variable with descriptive name :)